### PR TITLE
[sensors] Tag iio devices fo systemd, and require device4 in sensorfwd

### PR DESCRIPTION
**Pull request by piggz**
startup

### CODE REVIEW

File: sparse/etc/systemd/system/sensorfwd.service.d/90-boot-order.conf [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [Unit]
-Requires=systemd-modules-load.service
-ConditionPathExists=/dev/iio:device1
+Requires=systemd-modules-load.service sys-devices-0044:8086:22D8.0001-HID\x2dSENSOR\x2d200083.6.auto-iio:device4.device
+After=systemd-modules-load.service sys-devices-0044:8086:22D8.0001-HID\x2dSENSOR\x2d200083.6.auto-iio:device4.device
```
Comment (g7) [inline sparse/etc/systemd/system/sensorfwd.service.d/90-boot-order.conf]: This shouldn't be needed

File: sparse/etc/systemd/system/sensorfwd.service.d/90-boot-order.conf [hunk 2]
```diff
@@ -1,3 +1,3 @@
 [Unit]
-Requires=systemd-modules-load.service
-ConditionPathExists=/dev/iio:device1
+Requires=systemd-modules-load.service sys-devices-0044:8086:22D8.0001-HID\x2dSENSOR\x2d200083.6.auto-iio:device4.device
```
Comment (g7) [inline sparse/etc/systemd/system/sensorfwd.service.d/90-boot-order.conf]: We can avoid checking for `systemd-modules-load.service` now (but I'm OK leaving it there)

**Comment by g7** [comment]
Seems to work fine on device, the boot order seems finally correct :+1: 

Some minor nitpicks, but otherwise LGTM :100:

**Comment by piggz** [comment]
[boot5.txt](https://github.com/sailfish-on-latte/droid-config-latte/files/2974961/boot5.txt)

**Comment by piggz** [comment]
Rename to .svg, and view the now correct boot order

**Comment by g7** [comment]
+1, successfully tested the new changes on-device. Plotting the boot order produces an output comparable to yours.

Merging :+1:

**Pull request merged**